src/CMakeLists.txt

```diff
@@ -172,5 +172,6 @@ add_library(pgexporter_ext SHARED ${SOURCES})
 set_target_properties(pgexporter_ext PROPERTIES LINKER_LANGUAGE C VERSION ${VERSION_STRING}
                                      SOVERSION ${VERSION_MAJOR} PREFIX "")
 target_link_libraries(pgexporter_ext PUBLIC)
+target_link_libraries(pgexporter_ext PRIVATE ${EXTRA_LIBS})
```

Comment:
You need to change the include and library options

Reply:
You can borrow a lot from https://github.com/pgexporter/pgexporter/blob/main/src/CMakeLists.txt#L31